src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.props

```
@@ -134,6 +134,12 @@ Copyright (c) .NET Foundation. All rights reserved.
     <ToolDepsJsonGeneratorProject>$(MSBuildThisFileDirectory)GenerateDeps\GenerateDeps.proj</ToolDepsJsonGeneratorProject>
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(NETCoreSdkBundledMSBuildInformationProps)' == ''">
+    <NETCoreSdkBundledMSBuildInformationProps>$(MSBuildThisFileDirectory)..\..\..\Microsoft.NETCoreSdk.BundledMSBuildInformation.props</NETCoreSdkBundledMSBuildInformationProps>
```

I'm confused. Where is this file? Are you trying to target the `src` folder? Might be clearer to do:
```suggestion
    <NETCoreSdkBundledMSBuildInformationProps>$(RepoRoot)src\Microsoft.NETCoreSdk.BundledMSBuildInformation.props</NETCoreSdkBundledMSBuildInformationProps>
```

---

oh this is handy - where is this defined? I used this pattern because I thought I had seen this used in other places.

---

In Arcade. RepoLayout.props or somesuch.